Libraries/CustomComponents/Navigator/Navigator.js

```diff
@@ -736,6 +736,7 @@ var Navigator = React.createClass({
     if (transitionVelocity < 0 || this._doesGestureOverswipe(releaseGestureAction)) {
       this._transitionToFromIndexWithVelocity(transitionVelocity);
     } else {
+      this._handlePop();
```

Comment:
I probably wouldn't do it this way. This will theoretically start the transition twice, and `_handlePop` also sometimes calls pop on child navigators instead of itself

I have refactored this code already and I think I have this issue fixed. The new code will land soon and we can double check